Update howto-mfa-nps-extension.md

slbishop opened the pull request:
Made changes to clarify that users need to be "registered" for MFA, not "enabled" for MFA since "enabled" implies that they are enabled at the user level and will require MFA for all sign-ins and not just those dictated by conditional access and NPS extension.

azurecla commented:
Hi __@pemj__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Ja-Dunn commented:
@pemj - Hi, thanks for your contribution. We are in the process of refactoring the Azure technical articles, so we can’t merge this PR. When the refactoring is complete, we can give you a link to the new location of the article you updated so you can resubmit your update, if you want to.

ktoliver commented:
@pemj - Thanks for your contribution to the Azure docs. On November 16th, we announced the availability of our Azure technical documentation in a new location on [docs.microsoft.com](https://docs.microsoft.com). You can [read about it on our team blog](https://docs.microsoft.com/teamblog/docs-november-update). This change means the article source lives in a new GitHub repository. Because there are some changes in the source markdown as a result of the move, we can’t pull your contribution over to the new repo without creating a ton of merge conflicts. We invite you to resubmit the change (if you are willing) to the new version of the article at [https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-creation-via-arm](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-creation-via-arm). Thanks for your understanding.

Commit:
Update howto-mfa-nps-extension.md

Made changes to clarify that users need to be "registered" for MFA, not "enabled" for MFA since "enabled" implies that they are enabled at the user level and will require MFA for all sign-ins and not just those dictated by conditional access and NPS extension.
